### PR TITLE
Support separate GKE clusters for cpu and gpu workers

### PR DESCRIPTION
**Pull request by teetone**
Support separate GKE clusters for the CPU and GPU worker managers via Docker Compose.

### CODE REVIEW

**Review by epicfaace — APPROVED** [review]
let's merge this into the cs324 branch for now. not sure if we want it in master branch yet as it might complicate deployment to stanford

**Comment by teetone** [comment]
> let's merge this into the cs324 branch for now. not sure if we want it in master branch yet as it might complicate deployment to stanford

We can modify the start script for Stanford in the deployment repo.

File: codalab_service.py [hunk 1]
```diff
@@ -489,6 +474,21 @@ def has_callable_default(self):
...
+        ),
+        CodalabArg(
+            name=f'worker_manager_{worker_manager_type}_kubernetes_auth_token',
+            type=str,
+            help='Kubernetes cluster authorization token for the Kubernetes worker manager',
+        ),
+        CodalabArg(
+            name=f'worker_manager_{worker_manager_type}_kubernetes_cert_path',
+            type=str,
+            help='Path to the generated SSL cert for the Kubernetes worker manager',
```
Comment (epicfaace) [inline codalab_service.py:490]:
```suggestion
            help=f'Path to the generated SSL cert for the {worker_manager_type} Kubernetes worker manager',
```

File: codalab_service.py [hunk 2]
```diff
@@ -489,6 +474,21 @@ def has_callable_default(self):
...
         ),
+        CodalabArg(
+            name=f'worker_manager_{worker_manager_type}_kubernetes_cluster_host',
+            type=str,
+            help='Host address of the Kubernetes cluster for the Kubernetes worker manager',
+        ),
+        CodalabArg(
+            name=f'worker_manager_{worker_manager_type}_kubernetes_auth_token',
+            type=str,
+            help='Kubernetes cluster authorization token for the Kubernetes worker manager',
```
Comment (epicfaace) [inline codalab_service.py:485]:
```suggestion
            help=f'Kubernetes cluster authorization token for the {worker_manager_type} Kubernetes worker manager',
```

File: codalab_service.py [hunk 3]
```diff
@@ -489,6 +474,21 @@ def has_callable_default(self):
             type=int,
             default=1,
         ),
+        CodalabArg(
+            name=f'worker_manager_{worker_manager_type}_kubernetes_cluster_host',
+            type=str,
+            help='Host address of the Kubernetes cluster for the Kubernetes worker manager',
```
Comment (epicfaace) [inline codalab_service.py:480]:
```suggestion
            help=f'Host address of the Kubernetes cluster for the {worker_manager_type} Kubernetes worker manager',
```

**Comment by epicfaace** [comment]
@teetone , great, just updated in https://github.com/codalab/deployment/pull/206.

Can you also update cs324 (https://github.com/codalab/deployment/pull/205) to use the new args as well?

**Pull request merged**